plugins/woocommerce/client/admin/client/settings-payments/components/ellipsis-menu-content/ellipsis-menu-content.tsx

```diff
@@ -194,8 +194,6 @@ export const EllipsisMenuContent = ( {
 				>
 					<Button
 						onClick={ resetWooPaymentsAccount }
```

Comment:
I think this function name is now a little misleading, since it's not actually resetting the account yet, just showing the modal. Left a suggested refactor below and we could also remove the function:

```suggestion
						onClick={ () => { setResetAccountModalVisible( true ); } } 
```

Reply:
sounds good!

Reply:
done in https://github.com/woocommerce/woocommerce/pull/53590/commits/a329616a1d3c44e3d34ab7bd05d9eb07e5698e42!